src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -582,6 +582,8 @@ ImmutableArray<byte> ReturnValueMarshallingDescriptor
         /// TODO: Ideally we would expose INamespace on INamespaceTypeDefinition. Right now we can only get the qualified namespace name.
         /// </summary>
         INamespace ContainingNamespace { get; }
+
+        bool HasBody { get; }
```

Comment:
This method needs a doc comment. Probably something like, "Returns true if this method contains executable code, false otherwise."